docs/ordliste.md

```diff
@@ -0,0 +1,31 @@
+# Ordliste
+
+## Datamarkedsplassen 
+Vårt finn.no der produsenter viser og forvalter data og innsikt, mens konsumenter finner og ber om tilgang til data.
+
+### Dataprodukt 
+En samling av ett eller flere [datasett](dataprodukter/dataprodukt.md#hva-er-et-datasett) som teamet mener hører sammen. Dataproduktet inneholder en overordnet beskrivelse og har en eier med et tilhørende kontaktpunkt. [Les mer om dataprodukt](dataprodukter/dataprodukt.md).
+
+#### Datasett 
+En del av et dataprodukt. Består av data, metadata og kode. Data er i dag tilgjengelig i form av views/tabeller på BigQuery.
+
+### BigQuery
+Database hostet av Google som er tilgjengelig for team på GCP. [Les om hvordan man kan flytte data til BigQuery](dataprodukter/index.md#flytte-data-til-bigquery).
```

Comment:
Skulle man hatt noe om at BigQuery kan representeres som et datasett i Datamarkedsplassen? Så slipper man å lese igjennom hele ordlisten for å skjønne hvert ord.